src/styles.css

```
@@ -484,12 +484,12 @@ footer a:hover{text-decoration:underline}
 
   /* Grid */
   .grid{grid-template-columns:1fr;gap:10px;margin-bottom:40px}
-  .org-card{padding:14px;animation:none;opacity:1}
+  .org-card{display:flex;flex-direction:column;height:100%;padding:14px;animation:none;opacity:1}
   .org-card:hover{transform:none}
   .card-header-row{gap:10px}
   .org-logo{width:40px;height:40px}
   .org-name{font-size:13px}
-  .org-desc{font-size:12px;-webkit-line-clamp:2;line-clamp:2;display:-webkit-box;-webkit-box-orient:vertical;overflow:hidden}
+  .org-card{display:flex;flex-direction:column;height:100%;font-size:12px;-webkit-line-clamp:2;line-clamp:2;display:-webkit-box;-webkit-box-orient:vertical;overflow:hidden}
```

Stylelint (17.11.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)